cs3/app/registry/v1beta1/resources.proto

```diff
@@ -70,6 +70,11 @@ message ProviderInfo {
   // The action to be displayed to the user on the context menu.
   // By default this is "Open with".
   string action = 9;
+  // OPTIONAL.
```

Comment:
Even if already merged, I see this is de facto a breaking change, as you're going to use the `product_name` to identify the application "type": as such, this field should be `REQUIRED`.

Incidentally, Reva master does not take decisions based on the type (we just set all language headers for every app), and the CS3 wopiserver only has a few extra properties if the application is "Collabora". Nevertheless the new field makes perfect sense, to not risk inconsistencies with an admin-configurable (and user-display-oriented) field.

Reply:
Ok. That makes sense. It means that we already had some hidden differences for the different products (collabora, codiMD and so on). Now we are making that official. IMHO we can keep it optional to avoid the breaking change on Reva master.

Reply:
Well, I prefer to break Reva master now :) (those comments are anyhow advisory - unfortunately - meaning that the compilation goes through). At least to keep the API consistent, where I expect the HTTP app provider endpoint and the WOPI `/iop/open` one to change as well.